content/projects/gsoc/2023/index.adoc

```diff
@@ -1,4 +1,148 @@
 ---
-layout: redirect
-redirect_url: /projects/gsoc
+layout: project
+title: "Google Summer of Code in Jenkins"
+description: "Landing page for the Google Summer of Code program in the Jenkins project"
+section: projects
+tags:
+- gsoc2023
+sig: gsoc
+opengraph:
+  image: /images/gsoc/opengraph.png
+links:
+  gitter: "jenkinsci_gsoc-sig:gitter.im"
+  meetings: "/projects/gsoc/#office-hours"
 ---
+
+// image:/images/gsoc/jenkins-gsoc-logo_small.png[Jenkins GSoC, role=center, float=left]
+link:https://developers.google.com/open-source/gsoc/[Google Summer of Code]
+(GSoC) is a global, online program focused on bringing students and new contributors into open source software development. GSoC Contributors work with an open source organization on a 10-22 weeks programming project under the guidance of mentors.
```

Comment:
We generally prefer sentence per line formatting so that it is easier to review.  Optional
```suggestion
(GSoC) is a global, online program focused on bringing students and new contributors into open source software development.
GSoC contributors work with an open source organization on a 10-22 week programming project under the guidance of mentors.
```